### PR TITLE
Specify PBS Pro in blah.config (SOFTWARE-2628) 

### PR DESCRIPTION
**Pull request by brianhlin**
Clemson and Baylor are using PBS Pro and depending on how it's compiled, we can't automatically detect it from the `qstat -version`. Instead of automatically detecting this for the user, we should just have them configure it. This set of commits also adds a python library for loading the env vars from the `blah.config` so they can be more easily referenced by python scripts.

I did some light testing on a Fermicloud VM and verified that the submit files looked different depending on if I specified `pbs_pro=no` or `pbs_pro`=yes

### CODE REVIEW

File: src/scripts/pbs_status.py
Comment (djw8605) [inline src/scripts/pbs_status.py]: Just for sanity, could you set to set the string to "lower" before being compared.

**Review by djw8605 — CHANGES_REQUESTED** [review]
Just 1 change.

File: src/scripts/pbs_status.py
Comment (brianhlin) [reply src/scripts/pbs_status.py]: Done + more commits to fix files included in the blahp packaging

**Comment by djw8605** [comment]
Looks good to me.  Merge at will. 

-Derek

**Comment by brianhlin** [comment]
Looks like merging is blocked until you approve it :)

**Review by djw8605 — APPROVED** [review]

**Pull request merged**
